app/views/ExplainHowPartiesAreRelatedView.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components._
+@import viewmodels.InputWidth._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    govukTextarea: GovukTextarea,
+    paragraph: Paragraph,
+    caption: Caption,
+    heading: Heading,
+    cancelLink: CancelApplicationLink
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("explainHowPartiesAreRelated.title"))) {
+
+    @formHelper(action = routes.ExplainHowPartiesAreRelatedController.onSubmit(mode)) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @caption(messages("explainHowPartiesAreRelated.caption"))
+        @heading(messages("explainHowPartiesAreRelated.heading"))
+        @paragraph(Html(messages("explainHowPartiesAreRelated.para.1")))
+        @paragraph(Html(messages("explainHowPartiesAreRelated.para.2")))
+
+         @govukTextarea(
+            TextareaViewModel(
+                field = form("value"),
+                label = Label(None)
+                )
+                .withRows(15)
+                .withAttribute("maxlength","8167")
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
+    }
```

Comment:
I think this one is missing the cancel link?

Reply:
Oops, had that commit unpushed!